skyagi-web/src/routes/+page.server.ts

```diff
@@ -5,9 +5,9 @@ import type { PageServerLoad } from './$types';
 export const load: PageServerLoad = async ({ url, locals: { getSession } }) => {
 	const session = await getSession();
 
-	// if the user is already logged in return them to the account page
+	// if the user is already logged in return them to the dasnboard page
 	if (session) {
-		throw redirect(303, '/account');
```

Comment:
FYI, this line has similar change in: https://github.com/litanlitudan/skyagi/pull/110

Reply:
Thanks @tonyz0x0 . I can close my PR.